spatialprofilingtoolbox/db/scripts/retrieve_feature_matrices.py

```diff
@@ -30,12 +34,13 @@
 
     if args.database_config_file:
         database_config_file = abspath(expanduser(args.database_config_file))
-    if not exists(database_config_file):
-        raise FileNotFoundError(
-            'Need to supply valid database config filename: %s', database_config_file
-        )
+        if not exists(database_config_file):
+            raise FileNotFoundError(
+                f'Need to supply valid database config filename: {database_config_file}'
+            )
```

Comment:
Shouldn't everything below this line also be moved into the if statement? They require the `database_config_file` to exist or it'll error.

Reply:
I think the behavior I had in mind was that the first block is argument parsing and the rest assumes parsed args. That is, instead of indenting everything below to fit it inside the if statement, I'd rather add an `else` clause that handles the (similar but different) error that `args.database_config_file` isn't even defined.